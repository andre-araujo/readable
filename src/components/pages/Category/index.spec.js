import React from 'react';
import { shallow } from 'enzyme';

import Category from '.';

describe('Category', () => {
  it('should match snapshot', () => {
    expect(shallow(<Category />)).toMatchSnapshot();
  });
});
