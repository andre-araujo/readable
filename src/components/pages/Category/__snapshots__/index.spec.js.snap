// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category should match snapshot 1`] = `
<div>
  <h1>
    Category
  </h1>
</div>
`;
